crates/uv/tests/it/python_pin.rs

```diff
@@ -198,6 +199,131 @@ fn python_pin() {
     }
 }
 
+#[test]
+fn python_pin_use_global_if_no_local() -> Result<()> {
+    let context: TestContext = TestContext::new_with_versions(&["3.11", "3.12"]);
+
+    let xdg = assert_fs::TempDir::new().expect("Failed to create temp dir");
+    let uv = xdg.child("uv");
```

Comment:
Why do we need to create and set this if you're setting `XDG_CONFIG_HOME` in the test context? Can't we just pull the directory from there? (maybe you just added that after?)